docs/advanced/chaos-experimentation.md

```diff
@@ -0,0 +1,229 @@
+
+The framework needs below mentioned components to perform experiments starting from storing the data into the Postgres database for each incoming request all the way to pass the experiment values to the Envoys to inject faults.
+
+| Component Name                                | Description |
+| --------------------------------------------- | ----------- |
+| `clutch.module.chaos.experimentation.api`     | Module that supports CRUD API for managing experiments like Create, Get, List, Cancel, etc|
+| `clutch.module.chaos.serverexperimentation`   | Module which is responsible to processes the experiment data that it receives from database to be UI ready
+| `clutch.module.chaos.experimentation.xds`     | Module which runs Envoy xDS management server which is responsible to propagate experiment values to Envoys
+| `clutch.service.chaos.experimentation.store`  | Service that defines the data layer to perform all database operations for experiments  
+| `clutch.service.db.postgres`                  | Service to connect to Postgres database
+
+In order to use Chaos Experimentation Framework, **registration of all the above components is required**. 
+
+It is recommended to run Envoy xDS management server (`clutch.module.chaos.experimentation.xds`) on a separate host.    
+
+### Configuration
+
+#### Frontend
+
+The frontend of the framework is completely configurable. Below is a sample [frontend config](https://github.com/lyft/clutch/blob/6990b5aa8b1e6a47a33b28e2aaab9783e4e9d084/frontend/packages/app/src/clutch.config.js) which will show the list of experiments and as well as workflow to start/stop an experiment. 
+
+```"@clutch-sh/experimentation": {
+module.exports = {
+  ...
+  "@clutch-sh/experimentation": {
+    listExperiments: {
+      description: "Manage fault injection experiments.",
+      trending: true,
+      componentProps: {
+        columns: [
+          { id: "target", header: "Target" },
+          { id: "fault_types", header: "Faults" },
+          { id: "start_time", header: "Start Time", sortable: true },
+          { id: "end_time", header: "End Time", sortable: true },
+          { id: "run_creation_time", header: "Creation Time", sortable: true },
+          { id: "status", header: "Status" },
+        ],
+        links: [
+          {
+            displayName: "Start Server Experiment",
+            path: "/server-experimentation/start",
+          },
+        ],
+      },
+    },
+    viewExperimentRun: {},
+  },
+  "@clutch-sh/server-experimentation": {
+    startExperiment: {
+      componentProps: {
+        upstreamClusterTypeSelectionEnabled: true,
+      },
+      hideNav: true,
+    },
+  },
+```
+
+#### Backend Server
+
+Below configuration will spin up all the required modules and services to store the data coming from frontend into Postgres database.
+
+```yaml title="backend/clutch-config.yaml"
+modules:
+  ...
+  - name: clutch.module.chaos.experimentation.api
+  - name: clutch.module.chaos.lyftexperimentation
```

Comment:
`  - name: clutch.module.chaos.lyftexperimentation` is not part of OSS - let's remove it